### PR TITLE
implement direct message lookup API

### LINKED ISSUE #164: DM lookup
I'll implement DM lookup API

### PR DESCRIPTION
**Pull request by mmmommm**
resolve #164 

Impl lookup direct message API.

### CODE REVIEW

File: direct_message.go [hunk 1]
```diff
@@ -0,0 +1,112 @@
+package gotwtr
+
+type DMEventField string
+type EventTypes string
```
Comment (sivchari) [inline direct_message.go:4]:
```suggestion
```

File: direct_message.go [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+	DirectMessageFieldText             DMEventField = "text"
+	DirectMessageFieldEventType        DMEventField = "event_type"
+	DirectMessageFieldCreatedAt        DMEventField = "created_at"
+	DirectMessageFieldDMConversationID DMEventField = "dm_conversation_id"
+	DirectMessageFieldSenderID         DMEventField = "sender_id"
+	DirectMessageFieldParticipantIDs   DMEventField = "participant_ids"
+	DirectMessageFieldReferencedTweets DMEventField = "referenced_tweets"
+	DirectMessageFieldAttachments      DMEventField = "attachments"
+)
+
```
Comment (sivchari) [inline direct_message.go:17]:
```suggestion
type EventTypes string
```

Comment (mmmommm) [reply direct_message.go:16]: @sivchari 
sorry i'm too late.
please review again 🙇

**Comment by sivchari** [comment]
LGTM

**Comment by sivchari** [comment]
please rebase.

**Comment by sivchari** [comment]
kindly ping @mmmommm

**Pull request merged**